### ISSUE TITLE
clang-format C# interface brace wrapping ignores class brace wrapping setting

### ISSUE THREAD

**Issue by eazrael** [llvm/llvm-project#52674]
OS: Win10 64bit 
Clang-format version: 13.0.0, precompiled binary
Language C#

When configuring linebreak before the opening brace after a class name, interface classes to not adhere to this setting. My expection is to either having a new option for brace wrapping after interfaces or interface configuring adheres to the class brace wrapping setting. 

Example Code:
```
namespace a
{
    public interface I {}
    public class C {}
}
```
Output when formatted with clang-format.exe --style="{BreakBeforeBraces: Custom, BraceWrapping: {AfterClass: true}}":
```
namespace a {
public interface I {}
public class C
{
}
}
```
The option AfterObjCDeclaration: true also does not change formatting. 

For reference my full formatting settings: 
```
AlignAfterOpenBracket: DontAlign
AlignArrayOfStructures: Right
AllowAllArgumentsOnNextLine: false
AlignOperands: DontAlign
AllowShortBlocksOnASingleLine: Empty
AllowShortEnumsOnASingleLine: true
AllowShortFunctionsOnASingleLine: None
AllowShortIfStatementsOnASingleLine: Never
AllowShortLambdasOnASingleLine: All
AlwaysBreakTemplateDeclarations: No
# BreakBeforeBraces: Allman
BreakBeforeBraces: Custom
BraceWrapping:
  AfterCaseLabel: true
  AfterClass: true
  AfterControlStatement: true
  AfterEnum: true
  AfterFunction: true
  AfterNamespace: true
  AfterStruct: true
  BeforeCatch: true
  BeforeElse: true
  AfterObjCDeclaration: true
  BeforeLambdaBody: false
  BeforeWhile: false
  IndentBraces: false
BreakBeforeConceptDeclarations: false
ColumnLimit: 120
ContinuationIndentWidth: 4
IncludeBlocks: Merge
IndentWidth: 4
IndentRequires: false
NamespaceIndentation: All
SpaceAfterCStyleCast: true
SpaceBeforeParens: false
SortIncludes: CaseInsensitive
```

**Comment by mydeveloperday** [comment]
I believe this has been fixed between 13 -> (14,13.0.1) this was a regression between 12.0 and 13.0 caused by an AfterEnum change.

I think it was resolved by ed367b9dff10ee1df9ac1984eb2ad7544da7ab06

**Comment by mydeveloperday** [comment]
Marking this closed as a duplicate
